modules/ROOT/pages/lb-mapping-rules.adoc

```diff
@@ -8,165 +8,292 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
+----
+/organizations/{orgid}/loadbalancers
+----
 
-The external load balancer domain name depends on the unique name you assign to it. In the following mapping rule examples, the load balancer is `lb-demo`.
 
-By default, your load balancer listens to external requests on HTTPS and communicates internally with your worker over HTTP. If you configured your Mule application within the VPC to listen on HTTPS, make sure you set `upstreamProtocol` to `HTTPS` when creating the mapping list using the xref:anypoint-platform-cli.adoc#cloudhub-load-balancer-mappings-add[`load-balancer mappings add`] command.
+== Using a Literal 1:1 Mapping
 
-=== URL Mapping
+Instead of using a pattern, you can map literal app names.
+In this case, the rules must match the applications exactly.
 
-You can pass the app name as an input URI and map it directly to the app name in CloudHub:
+This rule maps the default load balancer `lb-demo.lb.anypointdns.net` directly to the `myApp.cloudhub.io` app in CloudHub:
 
-[%header,cols="10a,20a,20a,10a"]
+[%header,cols="10a,20a,20a,20a,10a"]
 |===
-|Rule # |Input URL 2+^| Output URL
-|   |    *URI*   |       *appName*   |   *appURI*
-| 0 | /{app}/    | {app}             | /
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+| 1 | / | myApp | / | http 
 |===
 
-This rule maps `lb-demo.lb.anypointdns.net/{app}` to `{app}.cloudhub.io`. `{app}` is a pattern for any application name you pass in the inbound request.
+This rule matches the `dev-misc-myhellotest-v1-myapp` app if the input URI is `/misc/myhellotest/v1/ping`, but not `/misc/myhellootest/v1/ping`:
 
+[%header,cols="10a,20a,20a,20a,10a"]
+|===
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+| 1 | /{layera}/{domaina}/v{versiona}/ | dev-{layera}-{domaina}-{versiona}-myapp | /api/ | http
+|===
 
-=== Host Mapping
 
-If you have a wildcard certificate (such as `*.example.com`), you can use the `subdomain` variable to map any subdomain:
+You can also use literal 1:1 mappings for multiple applications by changing the *Input Path* to be static instead of a defined pattern:
 
-[%header,cols="10a,20a,20a,10a"]
+[%header,cols="10a,20a,20a,20a,10a"]
 |===
-|Rule # |Input URL 2+^| Output URL
-|   |  *URI*   |       *appName*    |  *appURI*
-| 0 | /        | {subdomain}   | /
+| Rule # | *Input Path* | *Target App* | *Output Path* | *Protocol*
+| 1 | /v1/applicationX | application-x | /api/ | http
+| 2 | /v1/applicationY | application-y | /api/ | http
+| 3 | /v2/applicationx | application-x-v2 | /api/ | http
 |===
-This rule automatically maps any request passed to a subdomain of `example.com` to the corresponding `appName`:
 
-* Passing `api.example.com` would redirect to `api.cloudhub.io` +
-* Passing `application.example.com` is mapped to `application.cloudhub.io`.
 
-The same applies to the https://en.wikipedia.org/wiki/Subject_Alternative_Name[Subject Alternative Names] (SANs) of your SSL endpoints.
+== Using Patterns in Mapping Rules
```

Comment:
Shouldn't this topic come earlier in the set?
See earlier mention of my need to understand pattern vs. literal earlier.